### PR TITLE
Decouple Ansible deployment examples

### PR DESCRIPTION
**Pull request by sbesson**
Add the two Ansible examples decoupled as separate repositories: http://github.com/ome/ansible-example-omero-public-user and http://github.com/ome/ansible-example-omero-three-nodes. Also add the Docker example http://github.com/ome/docker-example-omero.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,10 +45,8 @@ Three nodes
 
 This is an example of installing PostgreSQL, OMERO.server and OMERO.web on separate nodes.
 
-- `cd three-nodes; ansible-playbook playbook.yml`
-
-Alternatively, use docker-compose to run a similar configuration in docker:
```
Comment (joshmoore) [inline README.md]: I assume this gets added back when the other repo is added?

Comment (sbesson) [reply README.md]: Yes. https://github.com/ome/docker-example-omero/ has been transferred to the organization. I can certainly add them as part of this PR as well.

File: README.md [hunk 2]
```diff
@@ -8,14 +8,17 @@ You can copy these and modify them for your own use.
 Quick start
 ===========
 
-- Install `ansible`: e.g. `pip install ansible`
-- Install roles: `ansible-galaxy install -r requirements.yml -p roles`
+- Install Ansible: e.g. `pip install ansible`
```
Comment (manics) [inline README.md]:
```suggestion
- Install Ansible: e.g. `pip install ansible>=2.6.16,<2.7`. Other versions of Ansible may work but are not routinely tested by us.
```

**Comment by sbesson** [comment]
I like the usage of `<img` to reduce the relative size of images in the README but I am failing to make this working with the headers. Suggestions on tags that can be used to workaround this issue welcome.

File: README.md [hunk 3]
```diff
@@ -5,18 +5,24 @@ This repository contains example playbooks for installing OMERO with [Ansible](h
...
+    `pip install docker-compose`
 
-Basics
-------
+Install OMERO.server and OMERO.web using one of these playbooks:
 
-![single node installation of OMERO.web, OMERO.server, and PostgreSQL](images/one-node.png "One-node OMERO")
+### Basics
+
+<img src="images/one-node.png" alt="One-node OMERO" width="30%" align="left"/>
```
Comment (manics) [inline README.md]:
```suggestion
<img src="images/one-node.png" alt="One-node OMERO" width="30%"/>
```

File: README.md [hunk 4]
```diff
@@ -25,24 +31,29 @@ installation blueprints for this configuration as well as more advanced ones:
  * [Public user](#public-user): allow public access to selected data without the need to log in
  * [Three nodes](#three-nodes): use more physical or virtual hardware to scale up your installation
 
-Public user
------------
+### Public user
+
+<img src="images/public-user.png" alt="OMERO with public user" width="30%" align="left"/>
```
Comment (manics) [inline README.md]:
```suggestion
<img src="images/public-user.png" alt="OMERO with public user" width="30%"/>
```

File: README.md [hunk 5]
```diff
@@ -25,24 +31,29 @@ installation blueprints for this configuration as well as more advanced ones:
...
+    cd ansible-role-omero-public-user
+    ansible-galaxy install -r requirements.yml
+    ansible-playbook playbook.yml
 
-![single node installation with public user enabled](images/public-user.png "OMERO with public-user")
+### Three nodes
 
-Three nodes
------------
+<img src="images/three-nodes.png" alt="Three-nodes OMERO" width="40%" align="left"/>
```
Comment (manics) [inline README.md]:
```suggestion
<img src="images/three-nodes.png" alt="Three-nodes OMERO" width="40%"/>
```

**Comment by sbesson** [comment]
I am fairly happy with https://github.com/ome/omero-deployment-examples/blob/fdc378c9182881fd3fd03b0cbb518f278d748eb5/README.md as an incremental step to mention Ansible/Docker with reasonably-sized images. I'll leave the websockets example as an RFE for someone else in a separate PR.

**Pull request merged**